ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.14.yaml

```diff
@@ -0,0 +1,225 @@
+base_images:
+  rhel-8-golang-1.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile
+  from: rhel-8-golang-1.20
+  to: oadp-operator
```

Comment:
just in this file is necessary to change, but would apply to to others for standardization

add branch (version) info to image name. The convention we follow is adding `-oadp-1.3`, but since it already have oadp in its name, I would go with only `-1.3` (resulting in `oadp-operator-1.3`)